service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -161,6 +166,13 @@ public ResponseEntity<RunSetListResponse> getRunSets(UUID methodId, Integer page
 
   @Override
   public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
+    Optional<UserStatusInfo> user = controllerUtils.getSamUser();
+    if (user.isPresent()) {
+      log.info("User ID: {}", user.get().getUserSubjectId());
```

Comment:
I assume this is here for now as a placeholder until your next ticket? We probably want to have some standard form (and maybe even utility functions) for logging these, like "User X did action Y on resource Z"